src/api/app/models/repository.rb

```diff
@@ -281,7 +281,7 @@ def copy_to(new_project)
 #
 #  id                  :integer          not null, primary key
 #  block               :string(5)
-#  linkedbuild         :string(8)
+#  linkedbuild         :string(9)
```

Comment:
it should be introduced by https://github.com/openSUSE/open-build-service/pull/10731 but just if the migration was tested before.